modules/wallet/scan.go

```diff
@@ -1,15 +1,20 @@
 package wallet
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/crypto"
 	"github.com/NebulousLabs/Sia/modules"
 	"github.com/NebulousLabs/Sia/types"
+	"github.com/NebulousLabs/bolt"
 )
 
 const scanMultiplier = 4 // how many more keys to generate after each scan iteration
 
+const outputSize = 350 // approx. size in bytes of an output and accompanying signature
```

Comment:
This is an excellent number to have.

Also, ouch. If the encoding had been better designed, it'd be closer to 100 bytes. The problem is all the 8-byte prefixes for empty arrays in the transaction object and signature object. I like it's like 10 each. I would consider that more a shortcoming of the transaction layout than the encoding scheme.